synapseclient/client.py

```diff
@@ -3786,10 +3918,21 @@ def getPermissions(
 
         return self.get_acl(entity=entity, principal_id=principal_id)
 
+    @deprecated(
```

Comment:
I am a bit confused looking at the message. Based on my understanding of the old `ger_permissions` method, it seems like it is calling`f"/entity/{entity_id}/permissions"` which is the same as the new `get_entity_permission`. So if we deprecate the old get_permission method, shouldn't that the method to replace it will be `get_entity_permission`?

Reply:
We are pointing users to use the method on the dataclass objects instead of the API calls. If we sent users to use the API calls - Then yes, that is the direct replacement, however, the intention is that they'll use `.get_permissions` or `.get_acl` on a class object.

Reply:
Got it. Thanks for explaining.